<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelItem</class>
 <widget class="QWidget" name="ModelItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>34</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>34</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>34</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="model_item_id">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="model_item_list">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="model_item_type">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
